warehouse/query-core/src/main/java/datawave/core/iterators/TermFrequencyOffsetIterator.java

```diff
@@ -0,0 +1,163 @@
+package datawave.core.iterators;
+
+import org.apache.accumulo.core.data.ArrayByteSequence;
+import org.apache.accumulo.core.data.ByteSequence;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Range;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.iterators.IteratorEnvironment;
+import org.apache.accumulo.core.iterators.SortedKeyValueIterator;
+import org.apache.accumulo.core.iterators.WrappingIterator;
+import org.apache.hadoop.io.Text;
+import org.apache.log4j.Logger;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+import static datawave.query.Constants.TERM_FREQUENCY_COLUMN_FAMILY;
+
+/**
+ * Alternate implementation of the {@link TermFrequencyIterator} that operates on precomputed fully-qualified column qualifiers
```

Comment:
Clarify comment? I'm not sure what a 'precomputed fully-qualified column qualifier is' - or what would make a column qualified `partially-qualified`.

Reply:
I'll clarify.